mitiq/calibration/settings.py

```diff
@@ -212,7 +212,8 @@ def make_problems(self) -> List[BenchmarkProblem]:
         for i, benchmark in enumerate(self.benchmarks):
             circuit_type = benchmark["circuit_type"]
             num_qubits = benchmark["num_qubits"]
-            depth = benchmark.get("circuit_depth")
+            # Circuits only generated when depth is specified
```

Comment:
Sorry if this is a nitpick, but I don't think this comment is correct. Circuits are generated regardless of whether we have the default passed in `dict.get`. We need the default so that the type-checker is happy.

Reply:
Good point, I changed this to `# Set default to return correct type`.